partials/layouts/about.php

```diff
@@ -1,313 +0,0 @@
-									'dropCap' => false,
-								),
-								array(),
-							),
-						),
-					),
-				),
-			),
-		),
-	);
-
-	$layouts[] = array(
-		'category' => 'about',
-		'label'    => __( 'About', 'go' ),
-		'blocks'   => array(
-			array(
-				'core/cover',
-				array(
-					'overlayColor' => 'tertiary',
-					'align'        => 'wide',
-				),
-				array(
-					array(
-						'core/heading',
-						array(
-							'align'     => 'center',
-							'content'   => __( 'Protecting yourself', 'go' ),
-							'level'     => 1,
-							'textColor' => 'primary',
-						),
-						array(),
-					),
-					array(
-						'core/paragraph',
-						array(
-							'align'     => 'center',
-							'content'   => __( 'Miller &amp; Cole is tremendously proud of the impact that we have made in helping our clients by providing quality legal services and exceptional service.', 'go' ),
-							'dropCap'   => false,
-							'textColor' => 'primary',
-						),
-						array(),
-					),
-				),
-			),
-			array(
-				'core/columns',
-				array(
-					'align' => 'wide',
-				),
-				array(
-					array(
-						'core/column',
-						array(),
-						array(
-							array(
-								'core/heading',
-								array(
-									'align'   => 'left',
-									'content' => __( 'Quality Results', 'go' ),
-									'level'   => 3,
-								),
-								array(),
-							),
-							array(
-								'core/paragraph',
-								array(
-									'content' => __( 'Our goal is to create assets from our clients’ innovations through patent, trademark and copyright law.&nbsp; We take great pride in providing quality trademark legal services and exceptional customer service every single day.', 'go' ),
-									'dropCap' => false,
-								),
-								array(),
-							),
-						),
-					),
-					array(
-						'core/column',
-						array(),
-						array(
-							array(
-								'core/heading',
-								array(
-									'align'   => 'left',
-									'content' => __( 'Experienced', 'go' ),
-									'level'   => 3,
-								),
-								array(),
-							),
-							array(
-								'core/paragraph',
-								array(
-									'content' => __( 'The attorneys at Miller &amp; Cole work as a team to exceed each of our clients’ expectations. We have 30+ years of high-level experience helping businesses protecting the time, money and resources spent developing ideas and inventions.', 'go' ),
-									'dropCap' => false,
-								),
-								array(),
-							),
-						),
-					),
-				),
-			),
-			array(
-				'core/image',
-				array(
-					'url'             => get_theme_file_uri( '/partials/layouts/images/2x3.jpg' ),
-					'alt'             => __( 'Image description', 'go' ),
-					'caption'         => '',
-					'linkDestination' => 'none',
-					'className'       => 'size-full',
-					'align'           => 'wide',
-				),
-				array(),
-			),
-			array(
```

Comment:
Oh dang. I just realized these layouts are going away. Historically the Layout Selector in CB has depended on these layouts from Go. The LS will be rather useless once GSE comes out. We should probably move these to CoBlocks, right? Otherwise, we should deprecate the control like Site Design - this time not because of compatibility but because of its value.

Reply:
Good call out! We can keep these in Go and I will restore them.